ember-bootstrap/package.json

```diff
@@ -39,7 +39,7 @@
     "release": "pnpm exec release-it"
   },
   "dependencies": {
-    "@ember/render-modifiers": "^2.0.0",
+    "@ember/render-modifiers": "^3.0.0",
```

Comment:
This would be a breaking change for every consumer relying on v2 due to another dependency.